src/PositionService.Business/Commands/PositionUser/EditPositionUserCommand.cs

```diff
@@ -60,46 +46,42 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(EditPositionUserRe
     {
       if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemovePositions))
       {
-        return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
+        return ResponseCreatorStatic.CreateResponse<bool>(HttpStatusCode.Forbidden);
       }
 
       ValidationResult validationResult = await _validator.ValidateAsync(request);
 
       if (!validationResult.IsValid)
       {
-        return _responseCreator.CreateFailureResponse<bool>(
+        return ResponseCreatorStatic.CreateResponse<bool>(
           HttpStatusCode.BadRequest,
-          validationResult.Errors.Select(e => e.ErrorMessage).ToList());
+          errors: validationResult.Errors.Select(e => e.ErrorMessage).ToList());
       }
 
-      OperationResultResponse<bool> response = new();
+      bool result = false;
 
       if (await _repository.DoesExistAsync(request.UserId))
       {
-        response.Body = request.PositionId.HasValue
+        result = request.PositionId.HasValue
           ? (await _repository.EditAsync(request.UserId, request.PositionId.Value)).HasValue
           : (await _repository.RemoveAsync(request.UserId, _httpContextAccessor.HttpContext.GetUserId())).HasValue;
       }
       else
       {
         if (!request.PositionId.HasValue)
         {
-          return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.BadRequest);
+          return ResponseCreatorStatic.CreateResponse<bool>(HttpStatusCode.BadRequest);
         }
 
-        response.Body = (await _repository.CreateAsync(_mapper.Map(request))).HasValue;
+        result = (await _repository.CreateAsync(_mapper.Map(request))).HasValue;
       }
 
-      if (response.Body)
-      {
-        //ToDo add clear cache user Position data by userId
+      if (result)
+      {
+        await _globalCache.RemoveAsync((await _repository.GetAsync(request.UserId)).PositionId);
       }
 
-      response.Status = response.Body 
-        ? OperationResultStatusType.FullSuccess
-        : OperationResultStatusType.Failed;
-
-      return response;
+      return ResponseCreatorStatic.CreateResponse<bool>(body: result);
```

Comment:
no need change the command and use ResponseCreator here

Reply:
done